airbyte-integrations/connectors/source-amazon-seller-partner/acceptance-test-config.yml

```diff
@@ -2,8 +2,7 @@ connector_image: airbyte/source-amazon-seller-partner:dev
 tests:
   spec:
     - spec_path: "integration_tests/spec.json"
-      backward_compatibility_tests_config:
-          disable_for_version: "0.2.26"
+          disable_for_version: "0.2.27"
```

Comment:
```suggestion
```

Reply:
Sorry I do not got what are you asking about.
This line is removed already too.
https://github.com/airbytehq/airbyte/pull/18283/commits/9cfdd44abe76342b4c8fec81937d8c2ba0b0610a